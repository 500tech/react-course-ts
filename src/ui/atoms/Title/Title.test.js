import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Title } from '.';

test('Nothing is broken', () => {
  const wrapper = shallow(<Title color="red">Meow</Title>);

  expect(toJson(wrapper)).toMatchSnapshot();
});

test('Color matches prop', ()  => {
  const wrapper = shallow(<Title color="red">Meow</Title>);

  expect(wrapper.find('h1').props().style.color).toBe('red');
});

test('Default color is correct', ()  => {
  const wrapper = shallow(<Title>Meow</Title>);

  expect(wrapper.find('h1').props().style.color).toBe('blue');
});