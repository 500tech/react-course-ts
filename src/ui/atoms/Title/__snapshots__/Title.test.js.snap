// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nothing is broken 1`] = `
<h1
  style={
    Object {
      "color": "red",
      "fontWeight": "bold",
    }
  }
>
  Meow
</h1>
`;
